src/Machine.js

```diff
@@ -222,6 +222,9 @@ class Machine {
       this._includeMacro(macro, context, buffer);
     } else {
       // source inclusion
+
+      // replace for compatibility with Windows
+      instruction.value = instruction.value.replace(/\\/g, "/");
```

Comment:
As an option you can backslash escape the special chars again. There are just few of them: \n, \r, \t, \b, \f